packages/trip-details/src/components/fares-v2-table.tsx

```diff
@@ -206,8 +205,8 @@ const FaresV2Table = ({
               {!Number.isNaN(originalAmount) &&
                 originalAmount > 0 &&
                 index > 0 && <TransferIcon size={16} />}
-              {price
-                ? renderFare(price?.currency?.code, price?.amount)
+              {legPrice
```

Comment:
add a comment that legprice exists when original amount exists. There's probably a lot of cleanup possible overall but we can put it as a todo for now